src/common/config_opts.h

```diff
@@ -1346,6 +1346,7 @@ OPTION(rbd_cache_max_dirty_age, OPT_FLOAT, 1.0)      // seconds in cache before
 OPTION(rbd_cache_max_dirty_object, OPT_INT, 0)       // dirty limit for objects - set to 0 for auto calculate from rbd_cache_size
 OPTION(rbd_cache_block_writes_upfront, OPT_BOOL, false) // whether to block writes to the cache before the aio_write call completes (true), or block before the aio completion is called (false)
 OPTION(rbd_concurrent_management_ops, OPT_INT, 10) // how many operations can be in flight for a management operation like deleting or resizing an image
+OPTION(rbd_max_iops, OPT_U64, 0) // the max iops an image can reach, 0 means unlimited. It can be adjusted by "rbd image-meta set conf_rbd_max_iops"
```

Comment:
Sure, we can rename it to rbd_max_write_iops, since we didn't limit read for this patch